build.sbt

```diff
@@ -9,6 +9,13 @@ licenses := Seq("Apache License, ASL Version 2.0" -> url("http://www.apache.org/
 scalaVersion := "2.13.5"
 crossScalaVersions := Seq("2.12.12", scalaVersion.value)
```

Comment:
Maybe we should remove those lines?

Reply:
You're right.
Remove this 👍

Reply:
I fix it d22e2f7f904491c3bc78283404933cfb0d9508a9 👍